.travis.yml

```diff
@@ -1,6 +1,12 @@
 rvm: 2.2.3
 sudo: false
 cache: bundler
+addons:
+  apt:
+    sources:
+    - elasticsearch-2.1
+    packages:
+    - elasticsearch
```

Comment:
I verified that ES 1.4.0 was being run despite setting `elasticsearch-2.1` as the source by running `curl localhost:9200` in `before_script`, if you would like to check my work.